### ISSUE TITLE
Release 4.4.2 - Release Candidate 1 - Installation metrics

### ISSUE THREAD

**Issue by wazuhci** [wazuh/wazuh#16949]
### Packages tests metrics information
|||
| :--                              | :--                                                             |
| **Main release candidate issue** | #16966                                     |
| **Main packages metrics issue**  | #16948                                                   |
| **Version**                      | 4.4.2                                                   |
| **Release candidate**            | RC1                                                      |
| **Tag**                          | https://github.com/wazuh/wazuh/tree/v4.4.2-rc1          |

---

### Packages used
- Repository: `packages-dev.wazuh.com`
- Package path: `pre-release`
- Package revision: `1`

---

| Arquitecture | Build |
| :--         | :--   |
| AMD64        | https://ci.wazuh.info/view/Tests/job/Test_install_tier/582/  |
| ARM64        | https://ci.wazuh.info/view/Tests/job/Test_install_tier/584/  |
| I386         | https://ci.wazuh.info/view/Tests/job/Test_install_tier/583/  |
| ARM32        | https://ci.wazuh.info/view/Tests/job/Test_install_tier/585/  |

---

| System              | AMD64 | ARM64 | I386 | ARM32 |
| :--                 | :--:  | :--:  | :--: | :--:  |
| Amazon Linux 1      | :green_circle:    | :white_circle:     | :white_circle:    | :white_circle:     |
| Amazon Linux 2      | :green_circle:    | :white_circle:     | :white_circle:    | :white_circle:     |
| CentOS 5            | :green_circle:    | :white_circle:     | :green_circle:   | :white_circle:     |
| CentOS 6            | :green_circle:    | :white_circle:     | :green_circle:   | :white_circle:     |
| CentOS 7            | :green_circle:    | :green_circle:    | :green_circle:   | :white_circle:     |
| CentOS 8            | :green_circle:    | :white_circle:     | :white_circle:    | :white_circle:     |
| Debian 7            | :green_circle:    | :white_circle:     | :green_circle:   | :white_circle:     |
| Debian 8            | :green_circle:    | :white_circle:     | :green_circle: | :white_circle:     |
| Debian 9            | :green_circle:    | :green_circle:    | :green_circle:   | :green_circle:    |
| Debian 10           | :green_circle:    | :white_circle:     | :green_circle:   | :white_circle:     |
| Debian 11           | :green_circle:    | :white_circle:     | :green_circle:   | :white_circle:     |
| Fedora 29           | :green_circle:    | :green_circle:    | :white_circle:    | :green_circle:    |
| Fedora 31           | :green_circle:    | :white_circle:     | :white_circle:    | :white_circle:     |
| Fedora 32           | :green_circle:    | :white_circle:     | :white_circle:    | :white_circle:     |
| Fedora 34           | :green_circle:    | :white_circle:     | :white_circle:    | :white_circle:     |
| Fedora 35           | :green_circle:    | :white_circle:     | :white_circle:    | :white_circle:     |
| Fedora 36           | :green_circle:    | :white_circle:     | :white_circle:    | :white_circle:     |
| openSUSE Tumbleweed | :green_circle:    | :white_circle:     | :white_circle:    | :white_circle:     |
| Oracle Linux 6      | :green_circle:    | :white_circle:     | :white_circle:    | :white_circle:     |
| Oracle Linux 7      | :green_circle:    | :white_circle:     | :white_circle:    | :white_circle:     |
| Oracle Linux 8      | :green_circle:    | :white_circle:     | :white_circle:    | :white_circle:     |
| Red Hat 6           | :green_circle:    | :white_circle:     | :white_circle:    | :white_circle:     |
| Red Hat 7           | :green_circle:    | :white_circle:     | :white_circle:    | :white_circle:     |
| Red Hat 8           | :green_circle:    | :white_circle:     | :white_circle:    | :white_circle:     |
| Red Hat 9           | :green_circle:    | :white_circle:     | :white_circle:    | :white_circle:     |
| Solaris 11          | :green_circle:    | :white_circle:     | :white_circle:    | :white_circle:     |
| Solaris 10          | :green_circle:    | :white_circle:     | :white_circle:    | :white_circle:     |
| Ubuntu Bionic       | :green_circle:    | :white_circle:     | :white_circle:    | :white_circle:     |
| Ubuntu Focal        | :green_circle:    | :white_circle:     | :white_circle:    | :white_circle:     |
| Ubuntu Precise      | :green_circle:    | :white_circle:     | :white_circle:    | :white_circle:     |
| Ubuntu Trusty       | :green_circle:    | :white_circle:     | :white_circle:    | :white_circle:     |
| Ubuntu Xenial       | :green_circle:    | :white_circle:     | :green_circle:    | :white_circle:     |
| Windows 2016   | :red_circle:    [*](https://github.com/wazuh/wazuh/issues/16949#issuecomment-1531486528)     | :white_circle:     | :white_circle:    | :white_circle:     |
| macOS 10.15         | :green_circle:    | :white_circle:     | :white_circle:    | :white_circle:     |

---

Status legend:

 
--

<p dir="auto"><g-emoji class="g-emoji" alias="black_circle" fallback-src="https://github.githubassets.com/images/icons/emoji/unicode/26ab.png"><img class="emoji" alt="black_circle" src="https://github.githubassets.com/images/icons/emoji/unicode/26ab.png" width="20" height="20"></g-emoji> <em>Not started: The tasks didn't start yet.</em><br>
<g-emoji class="g-emoji" alias="yellow_circle" fallback-src="https://github.githubassets.com/images/icons/emoji/unicode/1f7e1.png"><img class="emoji" alt="yellow_circle" src="https://github.githubassets.com/images/icons/emoji/unicode/1f7e1.png" width="20" height="20"></g-emoji> <em>In progress: The team is already working on it.</em><br>
<g-emoji class="g-emoji" alias="green_circle" fallback-src="https://github.githubassets.com/images/icons/emoji/unicode/1f7e2.png"><img class="emoji" alt="green_circle" src="https://github.githubassets.com/images/icons/emoji/unicode/1f7e2.png" width="20" height="20"></g-emoji> <em>Ready to review: The product owner must audit and validate the results.</em><br>
<g-emoji class="g-emoji" alias="white_circle" fallback-src="https://github.githubassets.com/images/icons/emoji/unicode/26aa.png"><img class="emoji" alt="white_circle" src="https://github.githubassets.com/images/icons/emoji/unicode/26aa.png" width="20" height="20"></g-emoji> <em>Skipped:  The task has been deemed not necessary for this stage.</em><br>
<g-emoji class="g-emoji" alias="brown_circle" fallback-src="https://github.githubassets.com/images/icons/emoji/unicode/1f7e4.png"><img class="emoji" alt="brown_circle" src="https://github.githubassets.com/images/icons/emoji/unicode/1f7e4.png" width="20" height="20"></g-emoji>  <em>Partially skipped:  Some parts of the task are skipped.</em><br>
<g-emoji class="g-emoji" alias="purple_circle" fallback-src="https://github.githubassets.com/images/icons/emoji/unicode/1f7e3.png"><img class="emoji" alt="purple_circle" src="https://github.githubassets.com/images/icons/emoji/unicode/1f7e3.png" width="20" height="20"></g-emoji> <em>Completed: Task finished. Nothing to do here.</em><br>
<g-emoji class="g-emoji" alias="red_circle" fallback-src="https://github.githubassets.com/images/icons/emoji/unicode/1f534.png"><img class="emoji" alt="red_circle" src="https://github.githubassets.com/images/icons/emoji/unicode/1f534.png" width="20" height="20"></g-emoji> <em>Completed with failures: Some issues were raised here.</em></p>

---

## Auditor's validation

In order to close and proceed with the release or the next candidate version, the following auditors must give the green light to this RC.

- [ ] @davidjiglesias 
- [x] @teddytpc1 

---

**Comment by fcaffieri** [comment]
## Window 2016 :red_circle: 

<details>
<summary>Error</summary>

```
08:35:39  ================================== FAILURES ===================================

08:35:39  ____________________________ test_check_log_errors ____________________________

08:35:39  install_agent_test.py:64: in test_check_log_errors

08:35:39      assert expected_error == True, assert_description['error']['exception_not_detected'] + " in line {}".format(line)

08:35:39  E   AssertionError: An error was found in the log in line 2023/05/02 11:34:37 rootcheck: ERROR: No winmalware file: './shared/win_malware_rcl.txt'

08:35:39  E     

08:35:39  E   assert False == True

08:35:39  E     -False

08:35:39  E     +True
```
</details>

Test url: https://ci.wazuh.info/job/Test_install/122622/console

**Comment by teddytpc1** [comment]
Hi @72nomada, @DFolchA.
Do you know the cause of the Windows errors?

```TXT
08:35:39  2023/05/02 11:34:37 rootcheck: ERROR: No winmalware file: './shared/win_malware_rcl.txt'
08:35:39  2023/05/02 11:34:37 rootcheck: ERROR: No winapps file: './shared/win_applications_rcl.txt'
```

**Comment by teddytpc1** [comment]
The error found in Windows is solved for `4.5.0`: https://github.com/wazuh/wazuh/issues/8309.
